host/traceh.c

```diff
@@ -67,88 +69,75 @@ void initialize_log_config()
         // inititalize if not already
         _log_level = _env2log_level();
         _log_file_name = getenv("OE_LOG_DEVICE");
+        _custom_log_format = getenv("OE_LOG_FORMAT");
```

Comment:
Instead of printing a newline after each custom log message, I am enforcing that the custom log format ends with a newline. Alternatively I could create a new longer string and append a newline at the end of it, but this seems cleaner. 

Please let me know if there is a better way to exit from here.

Reply:
"enforcing that the custom log format ends with a newline" sounds good to me.